src/interactives/global-climate/global-climate-1.json

```diff
@@ -11,6 +11,8 @@
     "Follow an energy packet or molecule of carbon dioxide to see what happens when there is more carbon dioxide in the atmosphere.",
     ""
   ],
+  "lang": "es",
+  "i18nMetadata": "locales/metadata/interactives/global-climate/global-climate-1.json",
```

Comment:
We don't want "lang": "es" in the English version of the interactive. 😉 It would switch all the default labels to Spanish. So this property ("lang") should not be in any of the English, original interactives. 

"i18nMetadata" property is correct and it should be there.